programs/validator-history/src/state.rs

```diff
@@ -524,6 +524,132 @@ impl ValidatorHistory {
     }
 }
 
+#[account(zero_copy)]
+pub struct ClusterHistory {
+    pub struct_version: u64,
+    pub bump: u8,
+    pub _padding0: [u8; 7],
+    pub history: CircBufCluster,
+}
+
+#[zero_copy]
+pub struct ClusterHistoryEntry {
+    pub total_blocks: u32,
+    pub epoch: u16,
+    pub padding: [u8; 122],
```

Comment:
can you spend 10 minutes brainstorming what else we might want to add here for future stakenet features? want to make sure this enough.

anything that'd be useful for aggregation?
- total emissions
- ...

Reply:
we could add a lot to this. and since it's a singleton, don't need to be as worried about size or cost since it's a one time payment
- total mev collected
- total epoch credits 
- total stake accounts
- total # of voting validators or unique validators on leader schedule
- last restart slot (is a sysvar rn, but could track restarts over time / factor that into scoring)

think I'll make it 256 bytes per entry